app/scripts/views/mainTemplate.js

```diff
@@ -207,7 +209,7 @@
           self_.samplesync = _counterCasesSync;
           if (self_.hasfilters) {
             self_.selectedsamples = _resultCounterSelectedCases;
-            self_.selectedpatients = _resultSelectedCases;
+            self_.selectedpatients = _selectedCasesByFilters;
```

Comment:
In this case, _resultSelectedCases will never be used. Should we also remove related code?